docs/pages/typography-helpers.md

```diff
@@ -49,6 +51,22 @@ Lighten up your headers by adding a class of `.subheader` to any header element.
 
 ---
 
+## Drop Caps
+
+Add a drop cap of 3 lines to the begining of a paragraph.
```

Comment:
i mean this type of thing (dropcaps) are mainly found in magazines and now in new websites too ..... i meant that .... rest on you to decide as always